Dockerfiles/Readme.md

```diff
@@ -1,5 +1,5 @@
 # octopusdeploy/octo
-A docker wrapped version of the popular [octo.exe](https://octopus.com/docs/api-and-integration/octo.exe-command-line) command line tool
+A docker wrapped version of the popular Octopus CLI, [octo](https://octopus.com/docs/api-and-integration/octo.exe-command-line)
```

Comment:
Note for future us to fix up that doc location

Reply:
added to https://trello.com/c/DySZV2WR/31-move-octo-docs-to-octopus-rest-api-octopus-cli-and-update-generator